### ISSUE TITLE
Feedback from devs and OSS tool teams for 'A devs guide to...'

### ISSUE THREAD

**Issue by Erioldoesdesign** [simplysecure/devs-guide-to#58]
Here is where devs and OSS tool teams can add feedback for the microsite https://simplysecure.github.io/devs-guide-to/ in a comment so that all feedback is collected in a single place ready to be read, understood, discussed and triaged by the 'devs guide to...' team.

----

Please add your feedback in an individual comment on this issue. You'll need to be logged into a github account in order to offer fedback and you can edit comments after you've posted. Our guiding questions are:

- What did you like?
- What did you not like?
- What was hard to understand?
- Where did you need more examples?
- What was missing?

**Comment by eloquence** [comment]
Looks great overall, thanks much for sharing it and soliciting feedback. I liked the "Five Whys" approach, definitely something I can see myself using in practice.

Structural feedback: I was a bit confused by the navigational bar at the top vs. the blog section on https://simplysecure.github.io/devs-guide-to/ , i.e. I found myself wondering how the blog and the guide content relate to each other. Perhaps headlines or other ways of separating "news" content from "guide" content could make that distinction clearer.

**Comment by kelseysmithdesign** [comment]
hi @eloquence , thank you so much for the feedback! It's really great to hear. Your structural feedback is very helpful - we definitely want to avoid confusion and I think other people have the same issue. We'll make some changes to that.

**Comment by SaptakS** [comment]
I think this is a great project idea with very important resources for developers. Few things that came to mind while going through the website I will try to note below:

- I think the user testing topics are really helpful. I think user testing is a super important part of development which often is ignored, so definitely good resources. I think it might also be useful to add resources about how to draw conclusions from user testing for better development, or what are the methods they can verify that the new solution after user testing is better than the old one.
- I think another resource that might be helpful for future along the same veins is "A devs guide to interacting (or collaborating) with designers".
- I love the design of the website, but I think there are certain accessibility issues that I noticed. A thorough accessibility audit of the website might be helpful. I know that the website is currently just using a theme, so improving accessibility might mean modifying the theme (or overriding the theme) a bit, but might be worth it.
- I agree with @eloquence that I was a bit confused about the structuring and browsing the different articles. The navigation bar is kind of a mix of article listing and navigation, which can be a little confusing. So better article listing/grouping might make it easier to follow.

**Comment by yannchwen-horizontal** [comment]
I echo the points @eloquence and @SaptakS have raised above, re: navigation/structure confusion, accessibility and additional resource ideas.

Some additional thoughts/elaboration I have:

1. On accessibility: the colours are hard to read for me, especially the colour of the post metadata agains the background (not enough contrast). Also, the body text font is hard to read, especially on mobile. I tried to review the site mostly on mobile (on a small iPhone) but found it too tiring/overwhelming to read and understand. I'd prefer a simpler sans serif font, which would still go with the tech theme/aesthetic but be more accessible. 

2. Alignment: the content's far left alignment is throwing me off on desktop, especially on a large monitor. There's almost 60% of negative space on the right, and my brain would prefer the content to be horizontally centered. 

3. Button UI: I'd appreciate it if the buttons ("read more") were a box/pill ie more elevated from the background, not just text with an arrow

4. Organizing post content more visually: As someone with ADHD, I'm having a hard time following the paragraphs/lists of text in each post. The images, emojis and headings help, but overall the body text is still hard to follow. I'd appreciate having more differences in font size for headings, maybe a different background colour for quote blocks (if the theme permits), and (if you have capacity) a more visual summary/overview (with post its? a mind map of sorts?) at the top of each post for the very long posts. 

5. Post titles: I appreciate that some titles clearly show what the post is about ... I'd love to see that in every title. Currently, the titles "Imakefoss Twitter takeover" and "Resource #2" don't tell me much about the content, and I have to read the blurb to understand. 

6. Small typo: In the website header, should it be "a dev's guide" with an apostrophe?

**Comment by n8fr8** [comment]
* What topics or sections were most useful? Why?

I really like the "recruiting testers" piece, which was a bit surprising for me. However, I think, given the difficulty at getting people to participate in anything outside of the normal routing these days, this feels very important. I think the tone of the example messages was well done, and having more of these for different styles of engagements would be great.

I also felt that the detail on the actual mechanics of how to conduct an effective user test was very helpful and useful. Actually setting up an environment that is conducive to documenting a user's typical interactions and habits is difficult. Having more resources related to the sample scripts seems like a good place to focus.

 * How could the resource microsite be improved (content or design)?

High level, I agree with some of the accessibility and readability concerns. I appreciate the attempt at having a style or look for the site, but having it fit more in with boring "developer documentation" may end up giving it more weight and authority. It feels like too much narrative text covering somewhat obvious points... I found myself skimming a lot of that since it was also a bit hard to read. Once it gets into specific tactics and examples, it feels more useful.

 * What was confusing or hard to understand?

Nothing was confusing or that difficult to understand. I found myself least interested in the longer narrative explainer text, and more drawn to the structured samples, five whys, etc.... I understand that user testing is important, but need some help with my game plan!

 * What guides or resources would you like to see added?

The Synthesis portion of this guide was the most novel for me, and something I need to think on more... perhaps more "what next?" resources, or examples of user testing report outcomes from other projects, could be inspiring and motivating.
